SonarQube.TeamBuild.PreProcessor/Resources.resx

```diff
@@ -205,21 +199,9 @@ The full path to a settings file can also be supplied. If it is not supplied, th
   <data name="SLAP_AdditionalFileNameMustBeSpecified" xml:space="preserve">
     <value>The Roslyn profile exporter returned an AdditionalFile that does not specify a file name. The AdditionalFile will be ignored.</value>
   </data>
-  <data name="SLAP_AnalyzerNotFound" xml:space="preserve">
```

Comment:
The SLAP_xxx resources are copies that weren't deleted when the code was moved in a previous commit.